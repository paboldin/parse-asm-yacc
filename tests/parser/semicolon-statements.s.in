	.ascii	"test;me";;; .ascii "or me"; .ascii	"hello there"
	somecode;
