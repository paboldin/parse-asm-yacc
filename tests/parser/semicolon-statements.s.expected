(DIRECTIVE_STRING)	.ascii(TOKEN)	"test;me"
(DIRECTIVE_STRING) .ascii(TOKEN) "or me"
(DIRECTIVE_STRING) .ascii(TOKEN)	"hello there"
(TOKEN)	somecode
section: name = .text, flags = x
	.ascii	"test;me";;; .ascii "or me"; .ascii	"hello there"
	somecode;
